doozer/doozerlib/backend/konflux_client.py

```diff
@@ -606,6 +583,11 @@ def _inner():
                                 else:
                                     creation_time = current_time
                                 age = current_time - creation_time
+
+                                if pod_phase == 'Pending' and age > datetime.timedelta(hours=1):
```

Comment:
Ahh I see that its here. Maybe we should move it to someplace more visible?

Reply:
Promoted to a method parameter to make it easier to trace and adjust in the future.